.github/workflows/ci.yml

```diff
@@ -89,6 +89,39 @@ jobs:
           git diff
           [[ -z "$(git status --porcelain)" ]]
 
+  build-arm64:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-20.04
```

Comment:
There are no arm machines on the list, all available machines are x64 based. 
I think GH only supports arm on self hosted runners.

Reply:
I was trying to work around that with uraimo/run-on-arch-action@v2.5.0, but I cannot get it working, so I assume we have to wait for gh actions to add support.

Reply:
Ah I see, it's using QEMU emulator. Might be easier to test that locally first.